2022/0xxx/CVE-2022-0265.json

```diff
@@ -16,8 +16,8 @@
                 "version": {
                   "version_data": [
                     {
-                      "version_affected": "<",
-                      "version_value": "5.1"
+                      "version_affected": "<=",
```

Comment:
@JamieSlome Shouldn't this be `=` rather than `<=` based on the thread in https://huntr.dev/bounties/d63972a2-b910-480a-a86b-d1f75d24d563/ ?

Reply:
@dbolkensteyn - you are right ❤️ Adjusting it now...